doc/user/release-notes.html

```diff
@@ -56,6 +56,8 @@ <h2>Release notes for Crux</h2>
 
 <h4>Minor changes</h4>
 <ul>
+  <li>01 Apr 2022: XCorr scoring implemented in C++, temporarily solved bug in threading issue: https://github.com/crux-toolkit/crux-toolkit/issues/407 
```

Comment:
XCorr scoring implementing in C++. This resolves a longstanding bug in our multithreading code.